src/test/resources/testBytes

```diff
@@ -0,0 +1 @@
+���
```

Comment:
What is this doing in the git repository? If it is an error, maybe remove it and add the filename to `.gitignore`

Reply:
Files are removed, an so is the the experimental code, that creates and tests them.